Userland/Games/MasterWord/WordGame.cpp

```diff
@@ -175,7 +175,7 @@ void WordGame::read_words()
     m_words.clear();
 
     auto try_load_words = [&]() -> ErrorOr<void> {
-        auto response = TRY(Core::File::open("/res/words.txt"sv, Core::File::OpenMode::Read));
+        auto response = TRY(Core::File::open("/usr/share/MasterWord/words.txt"sv, Core::File::OpenMode::Read));
```

Comment:
I thought we had some other use for `words.txt` (that is not `MasterWord` specific), but apparently not (or it was just planned).